src/charm.py

```diff
@@ -107,9 +109,6 @@ def _on_config_changed(self, event):
             event: The event triggered when the relation changed.
         """
         self.unit.status = WaitingStatus("configuring temporal")
-        self.ingress.update_config(
-            {"service-hostname": self.external_hostname, "tls-secret-name": self.config["tls-secret-name"]}
-        )
```

Comment:
Do you need to call _require_nginx_route here as well?

Reply:
No need, this is reflected in the `test_ingress` integration test.

Reply:
Interesting, I wonder how the nginx-route relation knows to refresh its config when the related application's config changes.

Reply:
I believe it relies on a `relation_changed` hook:

`
self._charm.framework.observe(self._charm.on[self._nginx_route_relation_name].relation_changed, self._config_reconciliation)
`